test/DynamoCoreTests/Nodes/DropDownTests.cs

```diff
@@ -98,6 +113,31 @@ public void OpenXmlDYNwithSelectionNode()
             Assert.AreEqual("1", node.GetType().GetProperty("SelectedString").GetValue(node));
         }
 
+        [Test]
+        public void GetSelectedStringFromItemShouldReturnString()
+        {
+            // Initialize the node
+            var node = new myDropDown();
+            node.PopulateItems();
+            //node.GetSelectedStringFromItem(node.Items.FirstOrDefault());
+            Assert.AreEqual("Monday", node.GetType().GetMethod("GetSelectedStringFromItem").Invoke(node, new object[] { node.Items.FirstOrDefault() }));
+        }
+
+        [Test]
+        public void PopulateItemsShouldNotChangeSelectedIndex()
+        {
+            // Initialize the node
+            var node = new myDropDown();
+            node.PopulateItems();
+            node.SelectedIndex = 3;
+
+            // Populate the items again without modifying the items
+            node.PopulateItems();
+
+            // Index should stay the same
+            Assert.AreEqual(3, node.SelectedIndex);
```

Comment:
This test is to valid that calling `PopulateItems` should not change the original `SelectedIndex`